scripts/generate-ssl-certs.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
+
+if [ ! -e server.js ]
+then
+	echo "Error: could not find main application server.js file"
+	echo "You should run the generate-ssl-certs.sh script from the main MEAN application root directory"
+	echo "i.e: bash scripts/generate-ssl-certs.sh"
+	exit -1
+fi
+
 echo "Generating self-signed certificates..."
+mkdir -p ./config/sslcerts
 openssl genrsa -out ./config/sslcerts/key.pem -aes256 1024
 openssl req -new -key ./config/sslcerts/key.pem -out ./config/sslcerts/csr.pem
 openssl x509 -req -days 9999 -in ./config/sslcerts/csr.pem -signkey ./config/sslcerts/key.pem -out ./config/sslcerts/cert.pem
 rm ./config/sslcerts/csr.pem
-# resolve issue with bad password...
-# Error: error:0906A068:PEM routines:PEM_do_header:bad password read
```

Comment:
@lirantal for some reason I get this error running with ssl after generating a cert with a password:
`Error: error:0906A068:PEM routines:PEM_do_header:bad password read`

my suspicion is that node is not asking me for the password when running `grunt prod`. I am running with node v0.12.5

to resolve, I needed to remove the password from the pem like so:
```
openssl rsa -in ./config/sslcerts/key.pem -out ./config/sslcerts/newkey.pem && mv ./config/sslcerts/newkey.pem ./config/sslcerts/key.pem
chmod 0600 ./config/sslcerts/key.pem ./config/sslcerts/cert.pem
```
not sure if anyone else has seen this, but I mentioned the a reference to another person solving this issue the same way:
http://blog.mgechev.com/2014/02/19/create-https-tls-ssl-application-with-express-nodejs/

should I open a ticket for this?